### DISCUSSION TITLE
Is it possible to filter available database schemas from command line connection?

### DISCUSSION THREAD

**Discussion by Vichman67** [dbeaver/dbeaver#22228] [Q&A]
Hello,

I'm using DBeaver Enterprise Edition 23.3.0 under Windows 10 22H2.

My question is: I'm trying to connect from the command line to an Oracle database having several schemas, but I'd like the new connection would have filtered the list of available schemas - similar to what you get from the user interface when configuring a connection: "Edit connection > General section > Filters > Schemas/users > and then you specify the schemas to include or exclude".

Is it possible? I've  not seen any command line option for that.

As an example, my command line is:

"%DBEAVER_DIR%\dbeaver.exe" -reuseWorkspace -bringToFront -con "name=MyConnection | driver=Oracle | host=MyHost | port=MyPort | database=MyDBInstance | user=MyUsername | password=MyPw | connect=true | openConsole=true"

I've seen there is an option to add to the -con parameters several "Advanced connection parameters" (e.g. prop.connectTimeout=30) but I can't guess if this will help me with my question...

TIA.

**Comment by LonwoLonwo** [comment]
Hello @Vichman67 

All available options are described here: https://github.com/dbeaver/dbeaver/wiki/Command-Line.

If there is no option in this list - you can create a separate feature request ticket for your case.
